css/style.css

```diff
@@ -1,5 +1,92 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@400;500;700&family=Poppins&family=Roboto:wght@300;400;700&display=swap');
+
+/* Primary colours */
+:root {
+  --black: #1c1a19;
+  --orange: #ff6b00;
+  --off_white: #fff5e1;
+  --white: #fff;
+  --grey: #3c3a39;
+  --light_gray: #dbd8d7;
+  --nav_height: 45px;
+}
+
 *,
 *::before,
 *::after {
   box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+  color: var(--white);
+}
+
+header {
+  color: white;
+}
+
+.toolbar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 24px 0 24px;
+  background: var(--grey);
+  font-family: 'Roboto', sans-serif;
+  height: var(--nav_height);
+}
+
+nav h1 {
+  font-size: 18px;
+  color: var(--white);
+}
+
+.headline-container h1 span {
+  display: block;
+}
+
+.headline-container {
+  background: var(--black);
+  background-image: url("../assets/img/header_mobile.png");
+  background-position: right bottom; /* Positioning */
+  background-size: 100px;
+  background-repeat: no-repeat; /* Prevent showing multiple background images */
+  height: calc(100vh - var(--nav_height));
+  padding: 80px 24px 0 24px;
+}
```

Comment:
- [ ] Please fix the background image is overlapping the text.
### **Take a look**
![confuse](https://user-images.githubusercontent.com/55596970/158943718-0878f3fe-8e23-47d5-b795-a650b8f63341.png)
### **See Figmal Design**
![image](https://user-images.githubusercontent.com/55596970/158943844-c4da1fae-b398-4f2b-b931-fbc5377853c1.png)